### ISSUE TITLE
build script should use equals signs to set -X

### ISSUE THREAD

**Issue by lvh** [rackspace/rack#339]
I currently get this warning while building:

```
link: warning: option -X github.com/rackspace/rack/util.Commit 11fec2be1b7ba3f8faab745dc577e9120200d52a may not work in future releases; use -X github.com/rackspace/rack/util.Commit=11fec2be1b7ba3f8faab745dc577e9120200d52a
link: warning: option -X github.com/rackspace/rack/util.Version 1.0.0-dev may not work in future releases; use -X github.com/rackspace/rack/util.Version=1.0.0-dev
```

**Comment by jrperritt** [comment]
That is known and expected. Having the equals sign there allows the script to work with all Go versions, whereas removing it only seems to work with Go 1.5.

**Comment by rgbkrk** [comment]
It was using the `=` form for about... 15 minutes until Jon wasn't able to build on Go 1.4 locally. We switched back to stay backwards compatible until (likely) 1.6.

I wonder if there's a way to suppress just this warning while letting others or build errors pass right on through.

**Comment by jrperritt** [comment]
Closed by #340
